hugegraph-store/hg-store-grpc/pom.xml

```diff
@@ -91,23 +91,23 @@
                     <pluginArtifact>
                         io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:${os.detected.classifier}
                     </pluginArtifact>
-                    <!--默认值-->
+                    <!--Default value-->
                     <protoSourceRoot>${project.basedir}/src/main/proto</protoSourceRoot>
-                    <!--默认值-->
+                    <!--Default value-->
                     <!--<outputDirectory>${project.build.directory}/generated-sources/protobuf/java</outputDirectory>-->
                     <outputDirectory>${project.basedir}/src/main/java</outputDirectory>
-                    <!--设置是否在生成java文件之前清空outputDirectory的文件，默认值为true，设置为false时也会覆盖同名文件-->
+                    <!--Set whether to clear the files in outputDirectory before generating java files, the default value is true, and it will also override files with the same name when set to false-->
```

Comment:
```suggestion
                    <!-- Set whether to clear the files in outputDirectory before generating java files, the default value is true, and it will also override files with the same name when set to false -->
```